apps/bfd-server-ng/src/main/java/gov/cms/bfd/server/ng/beneficiary/model/Beneficiary.java

```diff
@@ -1,46 +1,11 @@
 package gov.cms.bfd.server.ng.beneficiary.model;
 
-import gov.cms.bfd.server.ng.util.DateUtil;
 import jakarta.persistence.Entity;
 import jakarta.persistence.Table;
-import java.util.List;
 import lombok.Getter;
-import org.hl7.fhir.r4.model.Patient;
 
 /** Main entity representing the beneficiary table. */
 @Entity
 @Getter
 @Table(name = "valid_beneficiary", schema = "idr")
-public class Beneficiary extends BeneficiaryBase {
-
-  /**
-   * Transforms the beneficiary record to its FHIR representation.
-   *
-   * @return patient record
-   */
-  public Patient toFhir() {
```

Comment:
Moved this method to `BeneficiaryBase` to be able to use it from `BeneficiaryCoverage`